changelogs/unreleased/igrishnov/gh-7479-bug-fix-default-uri-tarantoolctl-connect.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/tarantoolctl
+
+* Made host=localhost by default for tarantoolctl connect. Now, when specifying only the port, the connection is made to localhost.
```

Comment:
Nit: Keep it within 80 symbols wide and leave an issue number (see other entries for examples).

Reply:
Fixed